src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { useColonyContext } from '~hooks';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+
+import { Heading3, Heading4 } from '~shared/Heading';
+import Button from '~v5/shared/Button';
+import Modal from '~v5/shared/Modal';
+
+const displayName = 'v5.common.Modals.InviteMembersModal';
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const MSG = defineMessages({
+  modalTitle: {
+    id: `${displayName}.modalTitle`,
+    defaultMessage: 'Invite members to the private beta',
+  },
+  modalDescription: {
+    id: `${displayName}.modalDescription`,
+    defaultMessage:
+      'You can invite up to 100 members to your Colony during the private beta using the invite link below.',
+  },
+  invitesUsed: {
+    id: `${displayName}.invitesUsed`,
+    defaultMessage: '{invitesUsed}/{invitesAvailable} invites used',
+  },
+  inviteLinkHeading: {
+    id: `${displayName}.inviteLinkHeading`,
+    defaultMessage: 'Your unique invite link:',
+  },
+  buttonText: {
+    id: `${displayName}.buttonText`,
+    defaultMessage: `{isCopied, select,
+      true {Link copied}
+      other {Copy link}
+    }`,
+  },
+});
+
+// @TODO: Add logic to get the real amount of invites available and used.
+const invitesAvailable = 100;
```

Comment:
Shall we hold out with this until this functionality is merged?

Reply:
Yup, I'll merge this PR as I'm a bit biased towards doing small PRs but as soon as that functionality is available I can come back in and make another PR to connect it.